guides/.vscode/asciidoctor.code-snippets

```diff
@@ -0,0 +1,33 @@
+// https://code.visualstudio.com/docs/editor/userdefinedsnippets
+{
+	"ifeval": {
+		"scope": "asciidoc",
+		"prefix": "ifeval",
+		"body": [
+			"ifeval::[\"{build}\" == \"$1\"]",
+			"$0",
+			"endif::[]"
+		],
+		"description": "Macro ifeval"
+	},
+	"ifeval satellite": {
+		"scope": "asciidoc",
+		"prefix": "ifsatellite",
+		"body": [
+			"ifeval::[\"{build}\" == \"satellite\"]",
+			"$0",
+			"endif::[]"
+		],
+		"description": "Macro ifeval with satellite predefined"
+	},
+	"ifeval foreman": {
+		"scope": "asciidoc",
+		"prefix": "ifforeman",
+		"body": [
+			"ifeval::[\"{build}\" == \"foreman\"]",
+			"$0",
+			"endif::[]"
+		],
+		"description": "Macro ifeval with foreman predefined"
+	}
+}
```

Comment:
Do we need to add a line here?

Reply:
Hmm this one was generated by the VSCode. Let me configure it: `files.insertFinalNewline` must be true.